src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  transition: transform 300ms;
+
+  &__image {
+    margin-bottom: 40px;
+    height: 134px;
+  }
+
+  &__name {
+    width: 100%;
+    color: $main-color;
+    font-size: 12px;
+    font-weight: 500;
+    font-family: inherit;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    font-family: inherit;
+    line-height: 14px;
+    width: 100%;
+    margin-bottom: 16px;
+    color: $second-color;
+  }
+
+  &__review {
+    height: 16px;
+    margin-bottom: 24px;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price {
+    display: flex;
+    width: 100%;
+    margin-bottom: 16px;
+    justify-content: space-between;
+    align-items: center;
+    height: 18px;
+
+    &-text {
+      font-size: 12px;
+      font-family: inherit;
+      color: $second-color;
+    }
+
+    &-sum {
+      font-size: 16px;
+      font-weight: 700;
+      font-family: inherit;
+      color: $main-color;
+      line-height: 18px;
+    }
+
+    &:hover &__name {
+      color: #34568b;
+    }
```

Comment:
this selector is incorrect but correct version is located below `&:hover .card__name`